xcode/configs/base-mac.xcconfig

```diff
@@ -1,3 +1,2 @@
-ARCHS = $(NATIVE_ARCH_ACTUAL)
-VALID_ARCHS = i386 x86_64
-MACOSX_DEPLOYMENT_TARGET = 10.8
\ No newline at end of file
+ARCHS = $(ARCHS_STANDARD)
```

Comment:
This is the new recommended setting. In our case, perhaps because this project is set to use the legacy build system, $NATIVE_ARCH_ACTUAL was producing a list of intel architectures instead of arm.